dev/check/shellcheck.sh

```diff
@@ -2,14 +2,19 @@
 
 echo "--- shellcheck"
 
-trap "echo ^^^ +++" ERR
-
-set -e
-
-cd "$(dirname "${BASH_SOURCE[0]}")"/../..
+cd "$(dirname "${BASH_SOURCE[0]}")"/../.. || exit
```

Comment:
The check would fail silently otherwise if it could not cd to that folder (as we removed the `trap`)

```suggestion
cd "$(dirname "${BASH_SOURCE[0]}")"/../.. || exit 1
```